doppler-kubernetes-operator.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: doppler-kubernetes-operator
+  version: 1.5.0
+  epoch: 0
+  description: ""
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/DopplerHQ/kubernetes-operator
+      tag: "v${{package.version}}"
+      expected-commit: a147cf3dbdf5ef517a73953e8d1f71baeafc4f24
+
+  - uses: go/bump
+    with:
+      deps: github.com/gogo/protobuf@v1.3.2 golang.org/x/crypto@v0.17.0 golang.org/x/net@v0.17.0 github.com/prometheus/client_golang@v1.11.1
+      go-version: "1.21"
+
+  - runs: |
+      set -x
+      CGO_ENABLED=0 GO111MODULE=on GOOS=$(go env GOOS) GOARCH=$(go env GOARCH)
+      # https://github.com/DopplerHQ/kubernetes-operator/blob/a147cf3dbdf5ef517a73953e8d1f71baeafc4f24/Dockerfile#L23C72-L23C211
+      go build -a --ldflags="-X \
+      github.com/DopplerHQ/kubernetes-operator/pkg/version.ControllerVersion=${{package.version}}" \
+      -o manager main.go
```

Comment:
we can use `go/build` pipeline here. example: https://github.com/mauricio-dc-chainguard/os/blob/b3ab2e6a64771e1ebc7e3f23b4069dd91bbbeac7/kuberay-operator.yaml#L24-L29